integration-tests/logging-custom-json/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we really need this separate module for the tests?
Can't they be added to the `deployment` module?

Reply:
@geoand moved this integration test to a unit test in the `deployment` module, if it's enough for you I'm good with it 🙂

Reply:
Thanks.